### PR TITLE
55773 : When clicking on the first push notif the Exo app is closed automatically

### PR DESCRIPTION
**Pull request by wbenabdessalem**
When user connect for the first time and click on the first push notif 

Actual behavior : 
the Exo app is closed automatically

Expected behavior : 
the message will be open

Problem: Code forced the data to be exist,In fact, the url value can be null.
Fix: Adding check for null

### CODE REVIEW

File: eXo/eXoAppDelegate.swift [hunk 1]
```diff
@@ -340,5 +340,4 @@ struct AppUtility {
         UIDevice.current.setValue(rotateOrientation.rawValue, forKey: "orientation")
         UINavigationController.attemptRotationToDeviceOrientation()
     }
-
```
Comment (rdenarie) [inline eXo/eXoAppDelegate.swift]: not needed modification

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you complete your PR desceription by explaining why the Exo app closes, and how you fix it ?
Thanks

File: eXo/eXoAppDelegate.swift (hunk 1 above)
Comment (wbenabdessalem) [reply eXo/eXoAppDelegate.swift]: > not needed modification

This just clean of code

Comment (rdenarie) [reply eXo/eXoAppDelegate.swift]: In fact, it add a modification in a file you didnt modify. So it complexify the PR review and in the future the commit read will more complex.
We should keep the commit as small as possible, and this line removing seems not necessary.
By removing the change, we simplify the commit and make it more readable.

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Review by rdenarie — CHANGES_REQUESTED** [review]
About your description change :

> Problem: Code forced the data to be exist.
> Fix: Adding check for null

Can you explain which data, which variable was possibly null ?

**Comment by wbenabdessalem** [comment]
Can you explain which data, which variable was possibly null ?

RE: The url value can be null.

**Comment by rdenarie** [comment]
Can you add it your comment in the description ? 
Only the description will be in the commit message. The idea is to keep the information when we will reread the commit message in few months

**Comment by wbenabdessalem** [comment]
> Can you add it your comment in the description ? Only the description will be in the commit message. The idea is to keep the information when we will reread the commit message in few months

Done

**Review by rdenarie — APPROVED** [review]

**Pull request merged**